cylc/flow/task_events_mgr.py

```diff
@@ -1608,6 +1607,72 @@ def _reset_job_timers(self, itask):
         # Set next poll time
         self.check_poll_time(itask)
 
+    @staticmethod
+    def process_execution_polling_intervals(
+        polling_intervals: List[float],
+        time_limit: float,
+        time_limit_polling_intervals: List[float]
+    ) -> List[float]:
+        """Create a list of polling times.
+
+        Args:
+            (execution) polling_intervals
+            (execution) time_limit
+            (execution) time_limit_polling_intervals
+
+        Examples:
+
+        >>> this = TaskEventsManager.process_execution_polling_intervals
+
+        # Basic example:
+        >>> this([40, 35], 100, [10])
+        [40, 35, 35, 10]
```

Comment:
I'm probably wrong, but I would have expected the output to be:

```
[40, 35, 25, 10]
```

Because:

* The polling interval was set as 40, 35
* To get from the last poll to the ETL we need to add 25 seconds - `100 - (40+35) = 25`
* The post ETL interval was defined as 10.

Which makes sense to me from the definition of execution time limit polling intervals:

> List of intervals after execution time limit to poll jobs.
>
> -- https://cylc.github.io/cylc-doc/stable/html/reference/config/global.html#global.cylc[platforms][%3Cplatform%20name%3E]execution%20time%20limit%20polling%20intervals

Reply:
According to Dave we don't poll on the ETL. I did ask

Reply:
Gotcha.